### PR TITLE
Changelog 5.0

### PR DESCRIPTION
**Pull request by mpenkov**
Hi @piskvorky,

I figured we should get the changelog out in advance, since it would take more time than we usually have for our meetings. It ended up being a good call, because the script didn't work immediately and I had to tweak some throttling parameters.

Anyway, please have a look if you can, delete/organize/sort stuff as you see fit.

I'll try to have a look during the week as well.

### CODE REVIEW

**Comment by piskvorky** [comment]
@mpenkov why is this 5.0.0? Since the `Removed functionality & deprecations` section is empty… maybe 4.2.0? Or 4.1.3?

**Comment by mpenkov** [comment]
Yes, 4.2.0 is also fine.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,23 +3,83 @@ Changes
...
+* [#3298](https://github.com/RaRe-Technologies/gensim/pull/3298): test and build wheels for Py3.{7,8,9,10}, by [@mpenkov](https://github.com/mpenkov)
+* [#3300](https://github.com/RaRe-Technologies/gensim/pull/3300): Fix code formatting for FT_CMD definition, by [@pabs3](https://github.com/pabs3)
+* [#3303](https://github.com/RaRe-Technologies/gensim/pull/3303): add GitHub URL for PyPi, by [@andriyor](https://github.com/andriyor)
+* [#3308](https://github.com/RaRe-Technologies/gensim/pull/3308): get rid of tox, build things via github actions directly, by [@mpenkov](https://github.com/mpenkov)
+* [#3318](https://github.com/RaRe-Technologies/gensim/pull/3318): Clean up evaluate_word_pairs code, by [@piskvorky](https://github.com/piskvorky)
+* [#3329](https://github.com/RaRe-Technologies/gensim/pull/3329): Check gallery up to date as part of CI, by [@mpenkov](https://github.com/mpenkov)
+
+### Minor improvements
+
+**I'm not sure if these belong here in the change log, as such changes aren't really visible to the regular users**
```
Comment (mpenkov) [inline CHANGELOG.md:61]: @piskvorky What about these issues?

Comment (piskvorky) [reply CHANGELOG.md:61]: Ah, missed that. I'll fix it during the release process.

**Pull request merged**